### DISCUSSION TITLE
GCP project

### DISCUSSION THREAD

**Discussion by Olisaemeka111** [cloudposse/community#10] [Free Tier Support] [answered]
Please I need help with this assignment.

## Task 1
Start a new Terraform working directory satisfying the following requirements
1. Provision a regional `private` K8s cluster on GKE
2. A dedicated service account should be used (instead of the default one), and the service account should be created as part of the script 
3. It should be creating a `new VPC` instead of using the default one
4. Subnet should be created in the `London` region, and the cluster should use this subnet. Make sure CIDR ranges used by the cluster are in the `RFC1918 24-bit block`
5. Create two node-pools, one with 3 nodes without auto-scaling, another with 0 node by default with auto-scaling enabled. Allow the auto-scaling node-pool to use preemptible nodes4.
6. Allow outbound internet access to the private cluster without assigning externalIP addresses to it

**Comment by osterman** [comment, marked as answer]
Sorry, we are unable to help with such a broad exercise. Also, our focus is on AWS and not GCP.
